src/features/move-columns/js/column-movable.js

```diff
@@ -306,6 +306,7 @@
             post: function ($scope, $elm, $attrs, uiGridCtrl) {
               function enableColumnMove(){
                   if ($scope.grid.options.enableColumnMoving && $scope.col.colDef.enableColumnMoving) {
+                    offAllEvents();
```

Comment:
Nit: This could be simplified. In all cases now you want to offAllEvents so you could start off by doing that and then in the if check just onDownEvents (else is no longer really needed)